.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: java
+sudo: false
 jdk:
-  - oraclejdk8
+  - openjdk8
+script: mvn clean install --batch-mode
```

Comment:
Is the above on line 5 related to the travis failure?

Reply:
No
I added it to check for failing tests and success generating Omod

Reply:
Our convention has been for the commit message to reflect the actual changes.